src/pool/poolClient.ts

```diff
@@ -444,6 +458,18 @@ export class Client {
     this.exchangeRateTable[l1TokenAddress] = { ...this.exchangeRateTable[l1TokenAddress], ...exchangeRateTable };
     return this.exchangeRateTable[l1TokenAddress];
   }
+  async resolveStakingData(
+    l1TokenAddress: string,
+    userState: Awaited<ReturnType<UserState["read"]>>
+  ): Promise<StakeData> {
+    assert(this.config.acceleratingDistributorAddress, "Must have the accelerating distributor address");
+    const contract = this.getOrCreateAcceleratingDistributorContract();
+    const { cumulativeBalance } = await contract.getUserStake(l1TokenAddress, userState.address);
+    return {
+      cumulativeBalance,
```

Comment:
id just return cumulative balance by itself rather than in an object if you dont care about anything else